app/controllers/lets_encrypt/verifications_controller.rb

```diff
@@ -17,7 +17,7 @@ def render_verification_string
     end
 
     def certificate
-      LetsEncrypt::Certificate.find_by(verification_path: filename)
+      LetsEncrypt.config.certificate_model.constantize.find_by(verification_path: filename)
```

Comment:
Prevent directly use `constantize` in each action, please implement a class method `LetsEncrypt.certificate_model` to get the model already constantize.